packages/ibm-products/src/global/js/hooks/useCreateComponentFocus.js

```diff
@@ -27,11 +27,52 @@ export const useCreateComponentFocus = ({
       await wait(10);
       elm.focus();
     };
+    // FUNCTION TO ENSURE THE ELEMENT TARGETED IS NOT CONTAINED IN AN ELEMENT MARKED 'INERT'
+    const isNotContainedInInert = (elm) => {
+      if (!elm) return false;
+      const inertParent = elm.closest('[inert]');
+      return (
+        !inertParent || (!inertParent.hasAttribute('inert') && !elm.disabled)
+      );
+    };
+
+    const getActiveSteps = () => {
+      const allSteps = Array.from(document.querySelectorAll(blockClass));
+      return allSteps.filter((el) => {
```

Comment:
What do you think about using `.find` here instead? There's only ever one active step at a time and then when you call it you don't have to grab the first item (`getActiveSteps()` rather than `getActiveSteps()[0]`).

Reply:
Yep, I can do that... probably singular-ize the name too since it should only return 1